Services/TermsOfService/classes/class.ilObjTermsOfServiceGUI.php

```diff
@@ -222,6 +254,7 @@ protected function settings(): void
         $this->showMissingDocuments();
 
         $form = $this->getSettingsForm();
-        $this->tpl->setContent($form->getHTML());
+        $this->components[] = $form;
+        $this->tpl->setContent($this->dic->ui()->renderer()->render($this->components));
```

Comment:
We still need a new `MessageBox`  (type: info) component with the current state determined in `ilAdministrationSettingsFormHandler::FORM_TOS` and an additional link targeting to `\ilObjUserFolderGUI::generalSettingsObject`.

Please remove the `case` in `\ilObjUserFolderGUI::addToExternalSettingsForm` afterwads and delete the `ilAdministrationSettingsFormHandler::FORM_TOS` usage.

Reply:
What message should be put inside the MessageBox? 
I am not quite sure what it is for

Reply:
![TEST-7-Nutzungsvereinbarungen](https://user-images.githubusercontent.com/2434906/187197572-16f8eea7-20d6-4e46-9881-56091ed03511.png)